mythril/laser/ethereum/call.py

```diff
@@ -197,10 +197,12 @@ def get_call_data(
     )
 
     uses_entire_calldata = simplify(
-        memory_size - global_state.environment.calldata.calldatasize == 0
+        memory_size == global_state.environment.calldata.calldatasize
     )
 
-    if uses_entire_calldata is True:
+    if (isinstance(uses_entire_calldata, bool) and uses_entire_calldata) or (
```

Comment:
`uses_entire_calldata` is always `Bool` right?

Reply:
Probably so. I fixed it. However in the JUMPI implementation it's checked for bool as well. Can you double check?

Reply:
Reverted, it seems that it breaks a test, so I guess it's not always Bool. I cannot verify because I cannot reproduce locally (don't have the right solidity version…)